docs/content/webcomponents.js

```diff
@@ -18,9 +18,18 @@ function FantomasSettingIcon(type) {
                 tooltip: "Changing the default of this setting is not recommended."
             }
             break;
+        case 'deprecated':
+            settingType = {
+                icon: "bi-exclamation-triangle-fill",
+                color: "orange-recommendation",
+                tooltip: "This setting is deprecated and will be removed in a future version."
+            }
```

Comment:
Open to discussion if there's a better way to display this, I just figured there should be _some_ sort of callout for deprecated settings.